project/Dependencies.scala

```diff
@@ -293,14 +293,17 @@ object Dependencies {
         jacksonCore,
         jacksonAnnotations,
         jacksonDatabind,
-        jacksonScala,
         jacksonJdk8,
         jacksonJsr310,
         jacksonParameterNames,
         jacksonCbor,
         lz4Java,
         Test.junit,
-        Test.scalatest)
+        Test.scalatest) ++
+    (if (getScalaVersion() == scala3Version)
+      Seq("com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0-rc1" % "test")
```

Comment:
Maybe leave a note that it's safe to mix jackson version in tests for scala3 compilation because blah blah.